quic/s2n-quic-transport/src/transmission/application.rs

```diff
@@ -130,6 +133,12 @@ impl<'a, S: Stream, Config: endpoint::Config> Normal<'a, S, Config> {
 
             self.path_manager.on_transmit(context);
 
+            // The default sending behavior is to alternate between sending datagrams
+            // and sending stream data. This can be configured by implementing a
+            // custom datagram sender and choosing when to cede packet space for stream data.
+            self.datagram_manager
+                .on_transmit(context, self.stream_manager);
```

Comment:
One thing we need to consider is allowing datagrams to be prioritized over control frames (ACK, HANDSHAKE_DONE, etc)